libqtile/widget/prompt.py

```diff
@@ -661,10 +668,56 @@ def cmd_info(self):
             active=self.active,
         )
 
+    def cmd_exec_general(
+            self, prompt, object_name, cmd_name, selector=None, completer=None):
+        """
+        Execute a cmd of any object. For example layout, group, window, widget
+        , etc with a string that is obtained from startInput.
+
+        Parameters
+        ==========
+        prompt :
+            Text displayed at the prompt.
+        object_name :
+            Name of a object in Qtile. This string has to be 'layout', 'widget',
+            'bar', 'window' or 'screen'.
+        cmd_name :
+            Execution command of selected object using object_name and selector.
+        selector :
+            This value select a specific object within a object list that is
+            obtained by object_name.
+            If this value is None, current object is selected. e.g. current layout,
+            current window and current screen.
+        completer:
+            Completer to use.
+
+        config example:
+            Key([alt, 'shift'], 'a',
+                lazy.widget['prompt'].exec_general(
+                    'section(add)',
+                    'layout',
+                    'add_section'))
+        """
+        try:
+            obj = self.qtile.select([(object_name, selector)])
+        except _SelectError:
+            logger.info("cannot select a object")
```

Comment:
I would raise this to `error` log level for all of these errors.

Reply:
I fixed it by cc676ed2141121dbfaeaeb78907084582fc5686e . thank you for your comment.